openpype/hosts/max/plugins/publish/extract_pointcloud.py

```diff
@@ -46,30 +40,30 @@ class ExtractPointCloud(publish.Extractor):
     families = ["pointcloud"]
 
     def process(self, instance):
+        self.settings = self.get_setting(instance)
         start = int(instance.context.data.get("frameStart"))
         end = int(instance.context.data.get("frameEnd"))
-        container = instance.data["instance_node"]
         self.log.info("Extracting PRT...")
 
         stagingdir = self.staging_dir(instance)
         filename = "{name}.prt".format(**instance.data)
         path = os.path.join(stagingdir, filename)
 
         with maintained_selection():
-            job_args = self.export_particle(container,
+            job_args = self.export_particle(instance.data["members"],
                                             start,
                                             end,
                                             path)
             for job in job_args:
-                rt.execute(job)
+                rt.Execute(job)
 
         self.log.info("Performing Extraction ...")
         if "representations" not in instance.data:
             instance.data["representations"] = []
 
         self.log.info("Writing PRT with TyFlow Plugin...")
         filenames = self.get_files(container, path, start, end)
-        self.log.debug("filenames: {0}".format(filenames))
+        self.log.debug(f"filenames: {filenames}")
 
         partition = self.partition_output_name(container)
```

Comment:
```suggestion
        partition = self.partition_output_name(instance,data["members"])
```
 self.get_partitions needs some rewrite too, otherwise it will break
```
    def get_partition(self, members):
        """
        Get Partition Value
        """
        for member in members:
            opt_list = self.get_operators(member)
            for operator in opt_list:
                count = rt.execute(f'{operator}.PRTPartitionsCount')
                start = rt.execute(f'{operator}.PRTPartitionsFrom')

                return count, start
```